### ISSUE TITLE
Add RSS feeds

### ISSUE THREAD

**Issue by durden** [Codrspace/codrspace#41]
We should add a url to get an RSS feed for a user.  We could also create a 'global' codrspace.com feed that takes the top up-voted posts (see #40) for the day and publish those in the global feed or something similar.

**Comment by szeitlin** [comment]
I just want to vote up this issue and ask if anybody ever did anything about it?

**Comment by durden** [comment]
No, there hasn't been anything done about this.  I think the best fix would just be to put a link at the top of the page somewhere instead of relying on the browser icon.
